tablelog.py

```diff
@@ -50,12 +50,10 @@ def get_sample_batch(self, epoch: int, model_training: bool,
             key = (epoch,i,split)
             # Populate positive example dictionaries
             self.table_pos_sent1_str[key], self.table_pos_sent2_str[key] = elmo_sentence_decode(pos_sent_1[i]), elmo_sentence_decode(pos_sent_2[i]) # get string representations of sentences
-            assert self.table_pos_sent1_str[key].split(' ')[pos_token_1[i].item()] == self.table_pos_sent2_str[key].split(' ')[pos_token_2[i].item()] # assert that the word is the same in both pos_sent1 & pos_sent2
```

Comment:
why remove this assertion? (1/2)

Reply:
To your [comment/requested change above](https://github.com/jscuds/rf-bert/pull/27#pullrequestreview-908179921), it fails when synonyms are used, because the words won't be the same.

I think I can fix it by ensuring the word is the same `or` the word is a synonym.

Reply:
oh that makes sense, no worries